src/game_player.cpp

```diff
@@ -640,6 +640,7 @@ void Game_Player::UnboardingFinished() {
 	Unboard();
 	if (InAirship()) {
 		SetDirection(RPG::EventPage::Direction_down);
+		SetSpriteDirection(RPG::EventPage::Direction_down);
```

Comment:
> Player sprite should face down after exiting airship 

This could cause issues when the Player has a the facing locked. Did you test this?

Reply:
It changes player direction even when facing locked flag is set. Player will be locked to down direction after landing airship.

Reply:
I added comments for this